modules/importer/src/runner/csaf/mod.rs

```diff
@@ -103,7 +103,7 @@ impl super::ImportRunner {
 
         Walker::new(source)
             .with_progress(WalkerProgress(progress))
-            .walk(filter)
+            .walk_parallel(10, filter)
```

Comment:
We should not do this. At least, not force it. Doing this with RH data, you will get IP blocked right away.

Reply:
agree with @ctron in that better horizontal scalability running multiple instances and multiple importer is what is most useful .... though for the single instance use case ... we can do this if we default to some reasonable value like 4 and control with env var (and document)

Reply:
I'd prefer to control this via the importer configuration. Which can be different per source. But yes.

Reply:
Fair points. I'll defer the configuration and revert the `walk_parallel`.